components/prize_pool/commons/prize_pool.lua

```diff
@@ -110,6 +110,12 @@ PrizePool.config = {
 			return Logic.readBoolOrNil(args.currencyrateperopponent)
 		end
 	},
+	currencyRoundPrecision = {
+		default = 2,
```

Comment:
what should we set as default here?

Reply:
Yeah let's keep 2 as that's what we have live right now. Even if I'm personally a bigger fan of 0.